### DISCUSSION TITLE
重构的时代来临 - 课堂笔记 2026年继续回顾，时代继续重构，不会回到从前

### DISCUSSION THREAD

**Discussion by guttasingapore** [RoadSideUniversity/ClubHouse2024#25] [Show and tell]
2023.11.10 路边社 economic topic discussion https://www.clubhouse.com/house/%E8%B7%AF%E9%82%8A%E7%A4%BE-economic-topics-discussion
时代距离重构很近了

我们现在处在哪里
where we are today, restructure start from destroy and damaging, establishing the equilibrium,  balance before start of construction 
旧大陆崩溃： EU? Russian related? Middle east? Far East?
Geographical analysis: US, EU, JP, China, new domain
债务问题: Debt & credit, US creator, 接近转折点？ how far and how it goes?
撕表：destory balance sheet BS (thanks XP's input), by 4: add new debt, sustain by paying interest, remove debit by destroy other's BS, creating new BS. Trouble makers & solution provider, visible hand & invisible hand
技术改变未来

**Comment by guttasingapore** [comment]
未来地缘政治冲突的可能性： 中亚？（Thanks Cabin's voice input),

**Reply by Lewisrxliu** [reply]
Piers Morgan

https://youtu.be/LoWViuG5VYs?si=97cahov9Y_0fX-X4

https://youtu.be/PmoI-RAaRu4?si=jcBHVEvIfGzr9ZSM

**Reply by ergatea** [reply]
加一段

https://x.com/piersuncensored/status/1724182924726067474?s=46&t=S5hiQ_NZYF9pk2232dK-iw

**Comment by guttasingapore** [comment]
俄乌战争是两大阵营都输不起的战争：能源要素？民族历史要素？
欧洲反复尝试影响融合重建的失败（after WW2)： 德俄联盟，东亚联盟，德国历史值得回顾（明年我一定要带孩子们去柏林看一下!)， 欧洲的脆弱和韧性， 
东亚风险非常高 - 能源依赖，地缘政治，制造业依赖，海洋文明的港口依赖， 亚洲内部的问题

**Reply by guttasingapore** [reply]
欧洲议会投票废除一票否决权
https://zhuanlan.zhihu.com/p/668700292

**Reply by guttasingapore** [reply]
设想：当世界没有俄罗斯的制约与参与，俄罗斯如果消失了，各个联盟各国家各民族如果自处相处，未来迫在眉睫的问题：粮食危机席卷非洲及中东？边户大帝国崩溃的各种危机？几十年前帝国实力资源外放的自掘坟墓？

**Reply by guttasingapore** [reply]
黎巴嫩2020-2023 一个典型可参考的残酷案例
2023 
https://news.un.org/zh/story/2023/10/1122752
http://www.news.cn/globe/2023-08/30/c_1310737804.htm
2022 
https://www.epochtimes.com/gb/22/4/6/n13698828.htm
2021 
https://www.bbc.com/zhongwen/simp/world-58977167
https://www.yicai.com/news/101110924.html
2020
https://www.sohu.com/a/412123079_352347

**Reply by Lewisrxliu** [reply]
> 设想：当世界没有俄罗斯的制约与参与，俄罗斯如果消失了，各个联盟各国家各民族如果自处相处，未来迫在眉睫的问题：粮食危机席卷非洲及中东？边户大帝国崩溃的各种危机？几十年前帝国实力资源外放的自掘坟墓？

https://zh.m.wikipedia.org/wiki/北穆亞隧道
2023.12.1 The SBU blew up a part of the railway track between Russia and China. The explosion happened last night on the Baikal-Amur highway, near in the Severomuysky Tunnel located in Buryatia. It is the main railway connection between Russia and China.
https://amp.cnn.com/cnn/2023/11/30/europe/russia-train-fire-ukraine-intl/index.html
https://www.dailykos.com/story/2023/11/30/2208856/-Ukraine-Invasion-Day-646-short-circuits-or-multiple-explosions-on-the-RU-Baikal-Amur-Mainline

**Reply by guttasingapore** [reply]
https://www.iea.org/reports/world-energy-outlook-2023
scenarios of excluding Russia

**Comment by BCzoro** [comment]
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/149653335/e307b747-5d26-49b0-981e-e527935616c7)

**Reply by guttasingapore** [reply]
好棒

**Reply by Lewisrxliu** [reply]
关于德国和欧洲，集中营，战前与战后 Dec2023欧洲游记达豪集中营参观有感
https://en.wikipedia.org/wiki/Dachau_concentration_camp
https://www.kz-gedenkstaette-dachau.de/en/historical-site/virtual-tour/

”Arbeit macht frei： Work Sets You Free“ - 1933-1945-present
“There is one path to freedom. Its milestones are: obedience, honesty, cleanliness, sobriety, diligence, orderliness, self-sacrifice, truthfulness, love of the fatherland.” -1939

![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/9760baeb-dc78-44bb-9d55-60ee0fe2faa7)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/3de7ddbe-a084-4e85-94cc-5c4be5c3f271)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/2fa896af-4048-46e0-936c-6a7d55639551)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/1d583318-ac88-40f5-8da4-1f4130cbe9f6)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/23d5b81a-2384-4fe8-8f9c-f19c0696d013)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/be03949a-8a6f-49c9-bee7-2f9c9f63a3c8)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/e2d5d385-9c1a-4882-b572-ad1c62f5452d)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/c7a0699a-548d-4c90-baf4-5ce56f92be20)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/ccf4d648-d8a1-457a-b880-1701876dea56)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/b0549999-b5bb-4576-8aa3-222332764c68)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/c136547f-4717-4406-884d-f9edc3d8c723)

**Comment by guttasingapore** [comment]
我们目前非常接近转折点， 未来高度不确定 agree today likely approaching to the future tipping point very closely，
美国政治过去的记录-抛弃盟友重建秩序
美国财政政策的两难- 全球债务，全球化财富规模化转移，
俄乌战争类似过去的苏阿战争的重演
美国最有能力搞内循环，欧洲和东方没有条件搞内循环

参考书单- 《交易的世界：金钱、权力与大宗商品交易商》 - 感谢老师们的推荐

**Reply by guttasingapore** [reply]
Bing老师的好问题： 关于战争红利 - why investor 无法获利, 
军工复合体通过国防承包和服务获利，公开市场无法代表大多数的players & close market

**Reply by Lewisrxliu** [reply]
Economists podcast - The Intelligence: higher-for-longer interest rates

Economists have stopped waiting for interest rates to drop because it doesn’t seem to be coming. The upward pressure on long-term bond yields suggests that this situation could last for a while. How should the world adjust? Israel’s economy might be in good enough shape to withstand the next few months, but a longer war won’t come cheap.
https://podcasts.google.com?feed=aHR0cHM6Ly9yc3MuYWNhc3QuY29tL3RoZWVjb25vbWlzdGFsbGF1ZGlv&episode=NjU0Y2IzZWY4MjkwYTEwMDEyYTE5YWU5

https://www.ey.com/en_us/strategy/macroeconomics/could-higher-for-longer-interest-rates-bring-lower-for-longer-growth

**Comment by Lewisrxliu** [comment]
感谢Bing老师的分享：
ZF 谈到的关于F-35评估的过程的纪录片：https://www.pbs.org/wgbh/nova/xplanes/about.html
做空
https://www.bloomberg.com/news/features/2023-08-06/how-much-did-hindenburg-make-from-shorting-adani-dorsey-icahn
亚洲金融危机
https://www.amazon.com/Chastening-Inside-Crisis-Financial-Humbled/dp/1586481819?nodl=1&dplnkId=e5b6c3e4-8fae-446b-8589-be95fe6197f4

感谢zuro老师的分享
https://www.stcn.com/article/detail/885855.html

**Comment by Lewisrxliu** [comment]
感谢Jing的分享
https://m.youtube.com/watch?si=hEzd0wuPOPvoFYos&v=gBcR8ceg5X8&feature=youtu.be

**Comment by guttasingapore** [comment]
影视业面对重构的时代，感谢Charlie老师的分享

AI挑战编剧，演艺，人类传统文化产业的强趋势
参考：好莱坞编剧与演员罢工
https://www.reuters.com/business/media-telecom/striking-actors-reach-tentative-agreement-with-hollywood-studios-end-strike-2023-11-09/
历史上最大的影视业是色情业
商业模式的大转折
人类文化面对的挑战 生存状态，伦理， 虚拟，人机接口nurolink, AI 等高科技对未来的挑战

**Comment by guttasingapore** [comment]
AI 替代底层劳动者 同时创造新机会，"天道损有余而补不足 人道损不足而补有余"， “天地不仁，以万物为刍狗；圣人不仁，以百姓为刍狗。”

**Comment by Lewisrxliu** [comment]
关于缅甸，历史，现状，军事，政治，感谢Carbin老师的知识普及，脉络清晰有画面感，感谢🙏

感谢bing老师分享 一篇老文章：http://www.cnd.org/cr/ZK02/cr134.hz8.html
文革知青在缅甸

**Reply by Lewisrxliu** [reply]
缅甸危机步步加深
https://asia.nikkei.com/Spotlight/Myanmar-Crisis
https://youtu.be/QXWO01a0LUs?si=byUttXytDUllFjUl

**Comment by ergatea** [comment]
彭博创新经济论坛 李显龙总理炉边对话完整版 https://www.youtube.com/watch?v=NxxaWgd-Hw4

**Reply by guttasingapore** [reply]
Yale-NUS College Geopolitics Conference Mar 22-24 2023 这个论坛更直接，当时我在现场
https://www.youtube.com/watch?v=hmPsAtknrW0
https://www.youtube.com/watch?v=q0MQoM1XJjw
https://www.youtube.com/watch?v=GvLhjyKFabY
https://www.youtube.com/watch?v=X9davY9Fa-0

**Comment by Lewisrxliu** [comment]
延伸关于重构的话题，偶然发现油管TED的这段演讲相当切题并且极具启发意义， global power by security order, economic order & digital order by technology superpower

Next global superpower by political scientist Ian Bremmer 2023
最后一句 we need to know,发人深省，也许注定是一厢情愿的幻想
https://youtu.be/uiUPD-z9DTg?si=nj9ljDluFhynJRHj

**Comment by ergatea** [comment]
政府需要面对科技公司的挑战

lewis ***@***.***>于2023年11月18日 周六22:53写道：

**Comment by Lewisrxliu** [comment]
感叹：”黑格尔说过，哲学就是哲学史。同样，美学就是美学史。” 中国就是中国史，”一切的历史都是当代史

**Comment by Lewisrxliu** [comment]
A sharing by Stephen Harper: politics and populism

https://www.prageru.com/presenters/stephen-harper
https://www.prageru.com/video/politics-and-populism

**Comment by Lewisrxliu** [comment]
重构是从多年前开始的。
记念。
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/1c678aa3-204f-4306-81b5-b1654cecfed5)

**Comment by Lewisrxliu** [comment]
USTR Releases Annual Report on China's WTO Compliance

https://ustr.gov/about-us/policy-offices/press-office/press-releases/2024/february/ustr-releases-annual-report-chinas-wto-compliance

**Comment by Lewisrxliu** [comment]
女性主义觉醒与后女性主义革命

18岁少年试图理解的新世界，会发生翻天覆地的变化： 讨论的重点在于 Why cyborg is our ontology; it gives us our politics?   ”By the late twentieth century, our time, a mythic time, we are all chimeras, theorized and fabricated hybrids of machine and organism- in short, cyborgs. The cyborg is our ontology; it gives us our politics.” Page 7 of 88
”到了二十世纪末，我们的时代，一个神话时代，我们都是嵌合体，是机器和有机体的理论化和虚构的混合体——简而言之，赛博格。赛博格是我们的本体；它给了我们我们的政治。” by google translation

https://zh.m.wikipedia.org/zh-sg/賽柏格宣言
https://en.m.wikipedia.org/wiki/A_Cyborg_Manifesto

**Comment by guttasingapore** [comment]
世界继续重构中，这可能是未来的几十年中永恒的话题
2025.12.6 
美国2025版国家安全战略刚刚发布，确认将战略重心重新转向西半球：
“门罗主义的特朗普延伸”？
承认贸易战的失败？
还是一个战略欺骗？

https://www.whitehouse.gov/wp-content/uploads/2025/12/2025-National-Security-Strategy.pdf

https://www.reddit.com/r/China_irl/comments/1pepm05/美国发布2025国家安全战略/

https://m.163.com/dy/article/KG3536VA0556C7JA.html?spss=adap_pc

**Reply by guttasingapore** [reply]
此时此刻的历史模版：沿袭Nicholas John Spykman的NSS2025，解读欧洲历史上30年战争（Dreißigjähriger Krieg）以及曾经定义现代欧洲国家的威斯特法利亚体系，让我不由自主想起了cyber老师

Reference 
1)
https://youtu.be/iUKbLU4CGRQ?si=cyPZ64U05sSeD11a

2)
America's Strategy in World Politics: The United States and the Balance of Power，Nicholas J. Spykman

https://dn790000.ca.archive.org/0/items/in.ernet.dli.2015.5673/2015.5673.Americas-Strategy-In-World-Politics.pdf

3)
https://zh.wikipedia.org/zh-sg/三十年戰爭
https://zhuanlan.zhihu.com/p/461078325
https://weread.qq.com/web/reader/1fc322c0813ab9c7fg013493

**Comment by guttasingapore** [comment]
2026年开年的DAVOS：时代此等重构，吾辈有幸见识，并身处其中

https://amp.dw.com/en/davos-germanys-merz-says-old-world-order-unraveling/a-75606240

https://www.bbc.com/news/articles/c301jgd1qj6o.amp

https://www.weforum.org/stories/2026/01/davos-2026-special-address-by-mark-carney-prime-minister-of-canada/

https://www.reuters.com/world/asia-pacific/boj-set-hold-rates-steady-election-yield-spike-cloud-outlook-2026-01-22/
